src/partials/contacts.html

```diff
@@ -0,0 +1,95 @@
+          <li>
+            <h4 class="contacts__weekdays">Monday - Friday</h4>
+            <p class="contacts__hours">06:00 AM - 10:00 PM</p>
+          </li>
+          <li>
+            <h4 class="contacts__weekdays">Saturday - Sunday</h4>
+            <p class="contacts__hours">08:00 AM - 16:00 PM</p>
+          </li>
+        </ul>
+      </div>
+      <ul class="contacts__details">
+        <li>
+          <a class="contacts__info" href="tel:+610 383 766 284">+61(0) 383 766 284</a>
+        </li>
+        <li>
+          <a class="contacts__info" href="mailto:+610 383 766 284">noreply@envato.com</a>
+        </li>
+      </ul>
+    </li>
+    <li class="contacts__container">
+        <span class="contacts__title contacts__title--foodtruck">FOOD TRUCK</span>
+        <span class="contacts__city">Chicago</span>
+        <p class="contacts__description">Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
+        <div>
+        <ul class="contacts__opening-hours">
+          <li>
+            <h4 class="contacts__weekdays">Monday - Friday</h4>
+            <p class="contacts__hours">06:00 AM - 10:00 PM</p>
+          </li>
+          <li>
+            <h4 class="contacts__weekdays">Saturday - Sunday</h4>
+            <p class="contacts__hours">08:00 AM - 16:00 PM</p>
+          </li>
+        </ul>
+      </div>
+      <ul class="contacts__details">
+        <li>
+          <a class="contacts__info" href="tel:+610 383 766 284">+61(0) 383 766 284</a>
+        </li>
+        <li>
+          <a class="contacts__info" href="mailto:+610 383 766 284">noreply@envato.com</a>
+        </li>
+      </ul>
+    </li>
+    <li class="contacts__container">
+      <span class="contacts__title contacts__title--cafe">CAFE</span>
+      <span class="contacts__city">Chicago</span>
+        <p class="contacts__description">Fusce ut velit laoreet, tempus arcu molestie vulputate</p>
+        <div>
+        <ul class="contacts__opening-hours">
+          <li>
+            <h4 class="contacts__weekdays">Monday - Friday</h4>
+            <p class="contacts__hours">06:00 AM - 10:00 PM</p>
+          </li>
+          <li>
+            <h4 class="contacts__weekdays">Saturday - Sunday</h4>
+            <p class="contacts__hours">08:00 AM - 16:00 PM</p>
+          </li>
+        </ul>
+      </div>
+      <ul class="contacts__details">
+        <li>
+          <a class="contacts__info" href="tel:+610 383 766 284">+61(0) 383 766 284</a>
+        </li>
+        <li>
+          <a class="contacts__info" href="mailto:+610 383 766 284">noreply@envato.com</a>
+        </li>
+      </ul>
+    </li>
+  </ul>
+  <div class="contacts__buttons">
+    <button class="btn btn--primary" button type="submit">Our Locations 
+        <svg width="8" height="10" class=" btn__icon btn__icon--primary">
+        <use href="images/icons.svg#icon-Vector-1">
+
+        </use>
+    </svg>
+</button>
+    <button class="btn btn--secondary" button type="submit">Franchise
```

Comment:
Przepraszam jeżeli się zbytnio czepiam, a już w szczególności o nazwy klas, ale nazywając button primary i secondary dajesz jakby info że jest to pierwszy i drugi przycisk na stronie, co źle opisuje ten przycisk. Lepiej użyj nazw "btn--our-locations" przykładowo. Wtedy wiadomo że to ten przycisk, a nie inny gdzieś na stronie. Twoja sekcja jest jedną z 8 więc nazwy muszą być dobrze opisane :)